server-spi-private/src/main/java/org/keycloak/migration/migrators/MigrateTo9_0_0.java

```diff
@@ -55,6 +56,22 @@ public void migrateImport(KeycloakSession session, RealmModel realm, RealmRepres
 
     protected void migrateRealmCommon(RealmModel realm) {
         addAccountConsoleClient(realm);
+
+        ClientModel accountClient = realm.getClientByClientId(Constants.ACCOUNT_MANAGEMENT_CLIENT_ID);
+        RoleModel viewAppRole = accountClient.addRole(AccountRoles.VIEW_APPLICATIONS);
+        viewAppRole.setDescription("${role_" + AccountRoles.VIEW_APPLICATIONS + "}");
+        LOG.debugf("Added the role %s to the '%s' client.", AccountRoles.VIEW_APPLICATIONS, Constants.ACCOUNT_MANAGEMENT_CLIENT_ID);
+        RoleModel viewConsentRole = accountClient.addRole(AccountRoles.VIEW_CONSENT);
+        viewConsentRole.setDescription("${role_" + AccountRoles.VIEW_CONSENT + "}");
+        LOG.debugf("Added the role %s to the '%s' client.", AccountRoles.VIEW_CONSENT, Constants.ACCOUNT_MANAGEMENT_CLIENT_ID);
+        RoleModel manageConsentRole = accountClient.addRole(AccountRoles.MANAGE_CONSENT);
+        manageConsentRole.setDescription("${role_" + AccountRoles.MANAGE_CONSENT + "}");
+        LOG.debugf("Added the role %s to the '%s' client.", AccountRoles.MANAGE_CONSENT, Constants.ACCOUNT_MANAGEMENT_CLIENT_ID);
+        manageConsentRole.addCompositeRole(viewConsentRole);
+        LOG.debugf("Added the %s role as a composite role to %s", AccountRoles.VIEW_CONSENT, AccountRoles.MANAGE_CONSENT);
+
+        RoleModel manageAccountRole = accountClient.getRole(AccountRoles.MANAGE_ACCOUNT);
```

Comment:
Manage account shouldn't be a composite role. Instead the rest endpoints should check for both manage account and manage consent roles. I believe this was already added